tests/hawc/services/nih/test_pubmed.py

```diff
@@ -0,0 +1,206 @@
+import os
+from unittest import TestCase
+
+import pytest
+
+from hawc.services.nih import pubmed
+
+PUBMED_API_KEY = os.environ.get("PUBMED_API_KEY")
+
+
+@pytest.mark.skipif(
+    PUBMED_API_KEY is None, reason="Test environment requires $PUBMED_API_KEY to be set."
+)
+class PubMedSearchTests(TestCase):
+    def setUp(self):
+        pubmed.settings.connect(PUBMED_API_KEY)
+        self.term = "science[journal] AND breast cancer AND 2008[pdat]"
+        self.results_list = [19008416, 18927361, 18787170, 18487186, 18239126, 18239125]
+
+    def test_standard_query(self):
+        self.search = pubmed.PubMedSearch(term=self.term)
+        self.search.get_ids_count()
+        self.search.get_ids()
+        self.assertEqual(self.search.request_count, 1)
```

Comment:
change to `assert self.search.request_count == 1`; you'll need to do this for the rest too